common/src/main/java/mca/entity/ai/brain/tasks/chore/ChoppingTask.java

```diff
@@ -107,6 +103,31 @@ protected void keepRunning(ServerWorld world, VillagerEntityMCA villager, long t
         super.keepRunning(world, villager, time);
     }
 
+    private final static int TREE_LOGS_MAX_HEIGHT = 8; //and 30(about logs) is max height of largest trees
```

Comment:
This I'll likely make configurable, since y'know _modded go brrrrrr_

Reply:
I just added some commits for Config.java
sorry, my English skill is not good lmao
is it right way?